### PR TITLE
[[FIX]] Add other fetch spec globals - Request, Response, Headers

### LINKED ISSUE #2582: Add other fetch globals: Response, Request, Headers
With the introduction of `fetch`, [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response), [`Request`](https://developer.mozilla.org/en-US/docs/Web/API/Request), and [`Headers`](https://developer.mozilla.org/en-US/docs/Web/API/Headers) are now globals in the browser.

### PR DESCRIPTION
**Pull request by lukeapage**
Fixes #2582

### CODE REVIEW

File: src/vars.js [hunk 1]
```diff
@@ -184,6 +185,8 @@ exports.browser = {
   removeEventListener  : false,
   resizeBy             : false,
   resizeTo             : false,
+  Response             : false,
+  Request              : false,
```
Comment (TheSavior) [inline src/vars.js]: `Request` would go before `requestAnimationFrame` if alphabetical, correct?

**Comment by TheSavior** [comment]
LGTM other than the alphabetical nitpick.

**Comment by marcoscaceres** [comment]
be nice to see this merged.

**Comment by jugglinmike** [comment]
I've re-ordered the properties and landed this at 07bb596c8e0303a97e60e036e6aedcabc8d24e2e. Thanks, @lukeapage!

**Pull request closed without merging**